.github/workflows/build_wheel_windows_arm64.yml

```
@@ -0,0 +1,53 @@
+name: Build Windows ARM64 Wheels
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
```

@NicolasHug would something like that addresses your concearns? I agree with @alinpahontu2912 that moving it to separate repo (just for Win-arm64) will create unnecessary complexity, and feel free to assume that total burden of reviewing an update PR will be on DevInfra teams sholders
```suggestion
  push:
    branches:
      - nightly
```

---

That reduces potential noise, so that's a decent compromise. As long as we can sandbox changes that are needed for windows support, and prevent the need for changing the rest of our scripts (i.e. avoiding interactions between linux/macos and windows jobs), that's OK.